### ISSUE TITLE
'Unable to serialize' exception in PhpStorm 2017.3 Public Preview

### ISSUE THREAD

**Issue by ptrkcsk** [zielu/GitToolBox#51]
Stack trace:

```
Unable to serialize GitToolBoxProjectSettings state: org/apache/commons/lang3/builder/EqualsBuilder
java.lang.NoClassDefFoundError: org/apache/commons/lang3/builder/EqualsBuilder
	at zielu.gittoolbox.config.CommitCompletionConfig.equals(CommitCompletionConfig.java:71)
	at java.util.AbstractList.equals(AbstractList.java:523)
	at com.intellij.openapi.util.Comparing.equal(Comparing.java:43)
	at com.intellij.util.xmlb.SkipDefaultsSerializationFilter.equal(SkipDefaultsSerializationFilter.java:64)
	at com.intellij.util.xmlb.SkipDefaultsSerializationFilter.equal(SkipDefaultsSerializationFilter.java:32)
	at com.intellij.util.xmlb.BeanBinding.serializeInto(BeanBinding.java:96)
	at com.intellij.util.xmlb.BeanBinding.serialize(BeanBinding.java:80)
	at com.intellij.configurationStore.XmlSerializer.serialize(xmlSerializer.kt:55)
	at com.intellij.configurationStore.XmlSerializer.serialize$default(xmlSerializer.kt:49)
	at com.intellij.configurationStore.SaveSessionBaseKt.serializeState(SaveSessionBase.kt:57)
	at com.intellij.configurationStore.SaveSessionBase.setState(SaveSessionBase.kt:29)
	at com.intellij.configurationStore.StateStorageManagerImpl$startExternalization$1.setState(StateStorageManagerImpl.kt:434)
	at com.intellij.configurationStore.ComponentStoreImpl.a(ComponentStoreImpl.kt:228)
	at com.intellij.configurationStore.ComponentStoreImpl.save(ComponentStoreImpl.kt:158)
	at com.intellij.openapi.components.impl.stores.StoreUtil.save(StoreUtil.java:49)
	at com.intellij.openapi.project.impl.ProjectImpl.save(ProjectImpl.java:349)
	at com.intellij.openapi.components.impl.stores.StoreUtil.saveDocumentsAndProjectsAndApp(StoreUtil.java:126)
	at com.intellij.openapi.application.impl.ApplicationImpl.saveAll(ApplicationImpl.java:1413)
	at com.intellij.openapi.vcs.actions.AbstractCommonCheckinAction.prepareRootsForCommit(AbstractCommonCheckinAction.java:100)
	at com.intellij.openapi.vcs.actions.AbstractCommonCheckinAction.actionPerformed(AbstractCommonCheckinAction.java:63)
	at com.intellij.openapi.vcs.actions.AbstractVcsAction.actionPerformed(AbstractVcsAction.java:48)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:220)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:237)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher$1.performAction(IdeKeyEventDispatcher.java:576)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.a(IdeKeyEventDispatcher.java:625)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:195)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:624)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.a(IdeKeyEventDispatcher.java:479)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:213)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:622)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:366)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by zielu** [comment]
Fixed in 173.1.1

**Comment by ptrkcsk** [comment]
Thanks!
